lib/public/views/About/About.js

```diff
@@ -11,19 +11,98 @@
  * or submit itself to any jurisdiction.
  */
 
-import { Observable } from '/js/src/index.js';
+import { getRemoteData } from '../../utilities/fetch/getRemoteData.js';
+import { Observable, RemoteData } from '/js/src/index.js';
 
 /**
- * Model representing handlers for homePage.js
+ * Model representing handlers for aboutPage.js
  */
-export default class Overview extends Observable {
+export default class AboutModel extends Observable {
```

Comment:
While this was the old convention of the project, the current convention is that we use named exports rather than default exports.
See `EosReportModel` on how to adapt